_components/header/header.html

```diff
@@ -32,7 +32,7 @@
     description: Display an extended header.
 changelog:
   key: component-header
-in_page_nav: false
+in_page_nav: "h2"
```

Comment:
I'd like to see a little more specificity in the variable name that's used. Currently, it's not immediately clear what the `in_page_nav` is supposed to do. Previously, it was a boolean, but now it accepts strings?

```suggestion
in_page_nav_headings: "h2"
```

Other suggestions:
1. `in_page_nav_targets`
2. `in_page_nav_controls` 
3. `in_page_nav_capture`

Reply:
Good point. I updated the name to `in_page_nav_headings` in [c3b1952](https://github.com/uswds/uswds-site/pull/2595/commits/c3b195241a5bb87dd49e12951847be491039de2f).

Note that the variable still accepts a value of `false`, which will remove it from the page. Is there any issue with this var accepting both strings and boolean values?

Reply:
Sounds good with the added clarity in variable name. We can follow-up in the future if this becomes an issue.

Thanks!